pkg/controller/direct/privilegedaccessmanager/entitlement_controller.go

```diff
@@ -73,6 +73,7 @@ func (m *entitlementModel) client(ctx context.Context) (*gcp.Client, error) {
 func (m *entitlementModel) AdapterForObject(ctx context.Context, reader client.Reader, u *unstructured.Unstructured) (directbase.Adapter, error) {
 
 	obj := &krm.PrivilegedAccessManagerEntitlement{}
+	fmt.Printf("Object received %v", u.Object)
```

Comment:
nit: cleanup?

Reply:
thanks for catching this. Even after grepping I dont know how i missed to catch this.